src/control-plane/backend/insert-admin-user.ts

```diff
@@ -33,7 +34,7 @@ export class AddAdminUser extends Construct {
     const putItemSdkCall: AwsSdkCall = {
       service: 'DynamoDB',
       action: 'putItem',
-      physicalResourceId: PhysicalResourceId.of('putItem-${props.userTable.tableName}'),
+      physicalResourceId: PhysicalResourceId.of(`AddAdminUserPut${generateRandomStr(8)}`),
```

Comment:
If you add a random string in every CDK synthsize, this resource always is updated in final CloudFormation. The result is the resource always is replaced in stack update. Is it expected behavior?

Reply:
All actions(create,update,delete) are set same PhysicalResourceId: `AddAdminUserCustomResource`